data/build.gradle

```diff
@@ -21,6 +21,10 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            buildConfigField "String", "ApiKey", APP_ID
```

Comment:
as you are repeating the line for release and debug, you can put it in the **defaultConfig** block

Reply:
Done